python/lsst/ip/isr/isrTask.py

```diff
@@ -287,7 +287,7 @@ def __init__(self, *, config=None):
         if config.doFlat is not True:
             self.prerequisiteInputs.discard("flat")
         if config.doFringe is not True:
-            self.prerequisiteInputs.discard("fringe")
+            self.prerequisiteInputs.discard("fringes")
```

Comment:
`self.prerequisiteInputs.remove` might be more suitable: it detects typos.

Reply:
That's what I said on slack!  All of these here should be `remove` instead of `discard`.

Reply:
I've switched them all, so this shouldn't come up again.